draft-terbu-oauth-sd-jwt-vc.md

```diff
@@ -197,18 +197,20 @@ SD-JWT VCs.
 
 #### `type` claim {#type-claim}
 
-This specification defines the JWT claim `type`. The `type` claim is used
-to express the type of the JSON object that is secured by the
-JWT. The `type` value MUST be a case-sensitive `StringOrURI` value.
+This specification defines the JWT claim `type`. The `type` claim value is the
+identifier for a pre-defined bundle of claims associated with certain
+policies for enrollment and issuance that the respective SD-JWT VC
+conforms to. The `type` value MUST be a case-sensitive `StringOrURI` value.
```

Comment:
```suggestion
This specification defines the JWT claim `type`. The `type` value
specifies the type of the VC represented by the SD-JWT VC. The
type determines the presence of JWT claims in the 
unsecured payload of the SD-JWT VC. The type also indicates which
JWT claims can have corresponding entries in the `_sd` claim and
are selective disclosable. The `type` value MUST be a case-sensitive
`StringOrURI` value.
```

Reply:
Rewording based on @awoie's proposal and incorporating Torsten's ideas.

```suggestion
This specification defines the JWT claim `type`. The `type` value
MUST be a case-sensitive `StringOrURI` value serving as an identifier
for the type of the SD-JWT VC. A type defines which claims MAY or MUST appear in the 
Unsecured Payload of the SD-JWT VC and whether they are
selectively disclosable or not. This specification does not define any `type` values; instead
it is expected that ecosystems using SD-JWT VCs define such values including
the semantics of the respective claims and associated rules (e.g., policies for issuing and
accepting credentials of a certain type).
```